app/services/users.js

```diff
@@ -0,0 +1,14 @@
+const db = require('../models');
+
+const createUser = body =>
+  new Promise((resolve, reject) => {
```

Comment:
You don't need to wrap this in a promise since the `User.newUser()` function already returns a promise